canonical-web-and-design/ubuntu.com
# Fix punctuation on Azure webinar

## Done

- Sentence cased the sub-head
- Remove the colon

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/ and http://0.0.0.0:8001/engage/azure-webinar
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See the changes
